eng/Versions.props

```diff
@@ -46,7 +46,7 @@
     <MicrosoftNETCorePlatformsPackageVersion>3.0.0-preview6.19229.9</MicrosoftNETCorePlatformsPackageVersion>
     <runtimenativeSystemIOPortsPackageVersion>4.6.0-preview6.19229.9</runtimenativeSystemIOPortsPackageVersion>
     <!-- Standard dependencies -->
-    <NETStandardLibraryPackageVersion>2.1.0-prerelease.19230.1</NETStandardLibraryPackageVersion>
+    <NETStandardLibraryPackageVersion>2.1.0-prerelease.19221.1</NETStandardLibraryPackageVersion>
```

Comment:
This is downgrading NETStandardLibrary package version and that would break official builds. How did you run the arcade update? Net standard should come from another channel.